cmd/mesh/manifest-migrate.go

```diff
@@ -15,23 +15,131 @@
 package mesh
 
 import (
+	"encoding/json"
 	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
 
+	"github.com/ghodss/yaml"
 	"github.com/spf13/cobra"
+
+	"istio.io/operator/pkg/apis/istio/v1alpha2"
+	"istio.io/operator/pkg/kubectlcmd"
+	"istio.io/operator/pkg/translate"
+	"istio.io/operator/pkg/util"
+	"istio.io/operator/pkg/version"
+)
+
+const (
+	defaultNamespace = "istio-system"
 )
 
-func manifestMigrateCmd(rootArgs *rootArgs) *cobra.Command {
+type manifestMigrateArgs struct {
+	// migrateDir indicates whether to migrate all values.yaml files in directory
+	migrateDir bool
+	// namespace is the namespace to get the in cluster configMap
+	namespace string
+}
+
+func addManifestMigrateFlags(cmd *cobra.Command, args *manifestMigrateArgs) {
+	cmd.PersistentFlags().BoolVarP(&args.migrateDir, "migrateDir", "r", false, "translate directory")
```

Comment:
"translate directory" -> "Directory tree containing Helm values YAML files. Only files with .yaml suffix are read."
Note that if you elect to remove --migrateDir, you should still create a filename flag here to explain that it can be a dir or file.

Reply:
done